src/inverseRobot.js

```diff
@@ -8,6 +8,36 @@
 
 function inverseRobot(robot) {
   // write code here
+  const ONE_VALUE_IN_OBJECT = 1;
```

Comment:
Consider using a more descriptive constant name such as 'INITIAL_COUNT' instead of 'ONE_VALUE_IN_OBJECT'